### PR TITLE
Master: pull request after DFTTK manuscript accepted

### PR DESCRIPTION
**Pull request by yiwang62**
pull request after DFTTK manuscript accepted using Master

### CODE REVIEW

File: dfttk/EVcheck_QHA.py [hunk 1]
```diff
@@ -219,31 +218,37 @@ def run_task(self, fw_spec):
...
-        modify_incar_params = self.get('modify_incar_params', {})
         modify_kpoints_params = self.get('modify_kpoints_params', {})
         site_properties = self.get('site_properties', None)
+
+        modify_incar_params = self.get('modify_incar_params', {})
+        powerups_options=modify_incar_params.get('powerups', None)
+
         override_default_vasp_params = self.get('override_default_vasp_params', {})
+        user_incar_settings = override_default_vasp_params.get('user_incar_settings',{})
+        powerups_options=user_incar_settings.get('powerups', powerups_options)
```
Comment (RushiGong) [inline dfttk/EVcheck_QHA.py:244]: powerups_options in 240

Comment (yiwang62) [reply dfttk/EVcheck_QHA.py:244]: Rushi: good catch

**Review by ShuangLin212 — COMMENTED** [review]
review

**Review by RushiGong — APPROVED** [review]
Review

**Review by ShuangLin212 — APPROVED** [review]

**Comment by yiwang62** [comment]
for DFTTK publication

**Pull request merged**